### PR TITLE
Faq own question fixes

### PR DESCRIPTION
**Pull request by StijnVrolijk**
Got complaints of a customer receiving the questions she asked herself in the OwnQuestion widget of the FAQ module. Went to investigate and came across this bug and a hardcoded reference to Fork CMS in the mail template.

### CODE REVIEW

File: src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php [hunk 1]
```diff
@@ -16,28 +16,28 @@
 use Frontend\Core\Engine\Navigation as FrontendNavigation;
 
 /**
- * This is a widget with the form to ask a question
+ * This is a widget with the form to ask a question.
```
Comment (WouterSioen) [inline src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php]: this has nothing to do with PSR2

File: src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php [hunk 2]
```diff
@@ -55,19 +55,19 @@ public function execute()
...
+     * Load the form.
      */
     private function loadForm()
     {
         // create form
         $this->frm = new FrontendForm('own_question', '#' . FL::getAction('OwnQuestion'));
-        $this->frm->addText('name')->setAttributes(array('required' => null));
-        $this->frm->addText('email')->setAttributes(array('required' => null, 'type' => 'email'));
-        $this->frm->addTextarea('message')->setAttributes(array('required' => null));
+        $this->frm->addText('name')->setAttributes(['required' => null]);
```
Comment (WouterSioen) [inline src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php]: We stil use the old array syntax (for consistency)

Comment (StijnVrolijk) [inline src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php]: I've remove the previous commit since it didn't actually do anything besides the array syntax

**Comment by jeroendesloovere** [comment]
:+1:

**Pull request merged**